package/tests/testthat/setup.R

```diff
@@ -12,6 +12,7 @@ Sys.setenv("QUARTO_TESTTHAT_DATA_PATH" = normalizePath(test_data_path))
 # (iv) run `devtools::test(filter = "test-<article_name>")`
 if_render_book <- TRUE
 if_render_articles <- FALSE
+if_test_plots <- TRUE # Additional option for optional skip of local plot tests
```

Comment:
If I allow testing plots it terminates early with a ton of error as follows. Could it be due to my setup or is it still outdated?

![Screenshot 2024-07-10 154934](https://github.com/insightsengineering/tlg-catalog/assets/11279768/bb2ce404-a9ee-4c60-8a90-305d3046af35)

Reply:
```
Error (test-graphs-other-bwg01.R:1:1): graphs/other/bwg01.qmd plot_v6 stable
Error in `ggplot2::ggsave(plot_file, data_snap[[i]], width = plot_dims[["width"]], 
    height = plot_dims[["height"]])`: Cannot find directory '_data/graphs/other'.
i Please supply an existing directory or use `create.dir = TRUE`.
```
Will try a fix ;)

Reply:
Fixed. should I remove the graphs anyway?

Reply:
removed so to be consistent w/ before